docs/source/commandline/commandline-all.rst

```diff
@@ -1301,39 +1753,62 @@ Core Sound Options
 
 **-samplerate** *<value>* / **-sr** *<value>*
 
-	Sets the audio sample rate. Smaller values (e.g. 11025) cause lower audio quality but faster emulation speed. Higher values (e.g. 48000) cause higher audio quality but slower emulation speed. The default is ``48000``.
+    Sets the audio sample rate.  Smaller values (e.g. 11025) cause lower audio
+    quality but faster emulation speed.  Higher values (e.g. 48000) cause higher
+    audio quality but slower emulation speed.
+
+    The default is ``48000``.
 
 .. _mame-commandline-nosamples:
 
 **-[no]samples**
 
-	Use samples if available. The default is ON (**-samples**).
+    Use samples if available.
+
+    The default is ON (**-samples**).
 
 .. _mame-commandline-volume:
 
 **-volume** / **-vol** *<value>*
 
-	Sets the startup volume. It can later be changed with the user interface (see Keys section). The volume is an attenuation in dB: e.g., "**-volume -12**" will start with -12dB attenuation. The default is ``0``.
+    Sets the startup volume. It can later be changed with the user interface
+    (see Keys section).  The volume is an attenuation in dB: e.g.,
+    "**-volume -12**" will start with -12dB attenuation.
+
+    The default is ``0``.
 
 .. _mame-commandline-sound:
 
 **-sound** *<``dsound``|``coreaudio``|``sdl``|``xaudio2``|``portaudio``|``none``>*
 
-	Specifies which sound subsystem to use. Selecting ``none`` disables sound output altogether (sound hardware is still emulated). On Windows, ``dsound``, ``xaudio2``, ``portaudio`` and ``none`` are available. On macOS, ``coreaudio``, ``sdl``, ``portaudio`` and ``none`` are available. On other operating systems, ``sdl``, ``portaudio`` and ``none`` are available. (Special build options allow ``sdl`` to be used on Windows, or ``portaudio`` to be disabled.)
+    Specifies which sound subsystem to use. Selecting ``none`` disables sound
+    output altogether (sound hardware is still emulated).
 
-	The default is ``dsound`` on Windows. On Mac, ``coreaudio`` is the default. On all other platforms, ``sdl`` is the default.
+| On Windows, ``dsound``, ``xaudio2``, ``portaudio`` and ``none`` are available.
+| On macOS, ``coreaudio``, ``sdl``, ``portaudio`` and ``none`` are available.
+| On other operating systems, ``sdl``, ``portaudio`` and ``none`` are available.  (Special build options allow ``sdl`` to be used on Windows, or ``portaudio`` to be disabled.)
+|
+| The default is ``dsound`` on Windows. On Mac, ``coreaudio`` is the default. On all other platforms, ``sdl`` is the default.
+|
 
-	On Windows and Linux, *portaudio* is likely to give the lowest possible latency, where on Mac *coreaudio* provides the best results.
+    On Windows and Linux, *portaudio* is likely to give the lowest possible
+    latency, where on Mac *coreaudio* provides the best results.
```

Comment:
There "where" in this line should be "whereas" or "while", amirite?

Reply:
Done.